File: Day 110/homework/hw2.py
# 2) დაწერეთ ფუნქცია, რომელიც მიიღებს მომხმარებლის სახელს. გვარსა და ასაკს. ფუნქციამ დააბრუნოს მომხმარებლის მონაცემები f სტრინგის გამოყენებით.




data = lambda name, surname, age: f"{name}, {surname}, {age}"

print(data("Gvanca", "Mazanishvili", 15))

File: Day 150/homework/hw2.js
    // 2) შექმენი ობიექტი user, კუთვნილებებით: name, age, city. თქვენი ამოცანაა დესტრუქტურიზაციის გამოყენებით ამოიღოთ ცვლადებში name, age და city.


const user  =  {
    name:  "gvanca",
    age: 16,
    city: "Rustavi"
}


const {name, age, city} = user;
console.log(name);
console.log(age);
console.log(city);
